ConeSearch.tex

```diff
@@ -638,6 +618,18 @@ \section{Changes from Previous Versions}
 
 \subsection*{Changes from WD-1.1-20200828}
 \begin{itemize}
+% 
+% remove the constraint of one RESOURCE only, clarifying it was there
+% only for the results
+%
+% added DALI MAXREC and RESPONSEFORMAT parameters, for response
```

Comment:
These items of the Change log are commented. Should some of them be un-commented now?

Reply:
Right, I'll update the changelog, maybe I'll wait the solutions to the above comments (i.e. the running meeting at least) before making this last update before merging.

Reply:
commit https://github.com/ivoa-std/ConeSearch/pull/64/commits/c29341446b051a57bfe4e3184f2d9662933dbf35 should have fixed this